c/pthread-C-DAC/LICENSE.txt

```diff
@@ -0,0 +1 @@
+email: hpcfte AT cdac.in
```

Comment:
@PhilippWendler Hi, I already emailed the author of the code, they said they don't have a license and I am free to use it as long as I don't change the header section.